docker/cheqd_node/node-runner.sh

```diff
@@ -36,7 +53,7 @@ if [[ ! -z ${P2P_SEND_RATE+x} ]]; then cheqd-noded configure p2p send-rate ${P2P
 set - # Disable ditailed command logging
 
 echo "Updating genesis"
-echo "$GENESIS" | base64 --decode > $CHEQD_ROOT_DIR/config/genesis.json
+echo "$GENESIS" > $CHEQD_ROOT_DIR/config/genesis.json
```

Comment:
If the common DIR is being used to pass in the genesis.json, there should be no need to read the genesis file, create an environment variable, and then go set it again back to a file. Either it's new node+volume in which case pass it in using tmpfs, or it's an existing volume in which the genesis file is already there.

Basically, let the user decide whether to pass in config use the compose file. If there's no genesis file, let the container panic and crash.

Reply:
Yup, I was choosing between getting rid of it at all and keep the ability to pass GENESIS as an environment parameter just for running docker container without composing.